index.test-d.ts

```diff
@@ -0,0 +1,16 @@
+import fastify from "fastify";
+import fastifyMysql, { MySQLPool } from ".";
+
```

Comment:
Could you also add some typings tests for plugin options and for imports?
Also looks like not all exported types are covered here.

Reply:
of cuz.